### ISSUE TITLE
[BUG] Print job stops randomly due to PR #2824

### ISSUE THREAD

**Issue by kisslorand** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2850]
### Description

With latest master print jobs are stopping randomly. I identified the culprit as PR #2824.

Others are also confirming this freeze issue:
https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/issues/2835#issuecomment-1737045630
> Display froze after about 15 seconds Restarted... lived 30 seconds this time (I have persistent info enabled) Reset to default values... I see different failure modes
> 
> 1. Screen freezes, doesn't respond anymore
> 2. Notification area is flashing, but screen doesn't respond anymore
> 3. Notification message changes when tapping the screen (from "busy" to "SD card inserted" and "SD card removed", cycles to theses 3 messages when I tap the screen, but I cannot exit

The author of the PR #2824 himself admits the presence of this bug here:
https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/issues/2835#issuecomment-1737157504
> The freeze should not be present but after some time TX and RX KPI drops to 0 in my TFTs.
> The bug is present even with unbuffered mode

### Steps to reproduce

1. Start a print
2. Pray
3. At some point print freezes...

**Expected behavior**
Print finishes successfully. 

**Actual behavior**
Print stops at random point.

### Hardware Variant
TFT: BTT TFT35 V3.0
MB: BTT SKR-2 with STM32F429VGT6

### TFT Firmware Version & Main Board Firmware details
TFT: latest master
Marlin: latest bugfix

### Proposed fix
I am asking @bigtreetech to urgently revert PR #2824 as it defeats the main purpose of a 3D printer, to print an object.

### Remarks
Those who do not have a backup or just want a more updated FW with print hesitation guard feature (not present in this FW) can download it from [here](https://github.com/kisslorand/BTT-TFT-FW).

**Comment by kisslorand** [comment]
I spent some more time on this bug and I thought it might be my rig, some memory corruption so I did a full reset, reinstalled everything. At first it seemed everything is back to normal the FW is working fine. After some repeated testing (with Advanced_OK on and off) suddenly the print stopped, there was no advancement of the progress (height, layer number, progress bar), only the timers were working. I didn't made any video because it might have seen as I am printing some long lines and it is normal for a period the height and layer number to stay constant. In fact I was printing a cylinder in vase mode without ARC, many many tiny segments at a high speed so height and layer number changes very rapdly during a normal print.
 After a while a more interesting thing happened, "No printer attached!" message appeared and the print started at an insane speed. I managed to capture it on video.
https://drive.google.com/file/d/14O3ADOWLXOJUiEAoc3cSpt4pknTv5xDG/view?usp=drive_link

**Comment by kisslorand** [comment]
OK, new data.
After several attempts I could reproduce the issue even with PR #2824 reverted, so I take back that  PR #2824 is the culprit.

If in the future anyone else reports this issue than we already know where NOT to look to, it's not  PR #2824.
I will close it know as I highly doubt a regular user would stress the TFT as I did to bring this bug to the surface.

I am sorry if I caused trouble to anyone with this.
